suntime/suntime.py

```diff
@@ -21,92 +24,96 @@ def __init__(self, lat, lon):
 
         self.lngHour = self._lon / 15
 
-    def get_sunrise_time(self, date=None, tz=None):
+    def get_sunrise_time(self, at_date=date.today(), time_zone=timezone.utc):
         """
-        :param date: Reference date. datetime.now() if not provided.
-        :param tz: pytz object with .tzinfo() or None
+        :param at_date: Reference date. datetime.now() if not provided.
+        :param time_zone: pytz object with .tzinfo() or None
         :return: sunrise datetime.
         :raises: SunTimeException when there is no sunrise and sunset on given location and date.
         """
-        date = datetime.datetime.now() if date is None else date
```

Comment:
FYI changing this is a regression: the today value is captured when the module is imported and the function is defined, then it stays the same as long as the process runs. Any web app for example using this function with the default value would get incorrect results after some time.

(+ also a doc issue: mention of now is not accurate, the value is a date not a datetime)